go.mod

```diff
@@ -516,7 +516,7 @@ replace (
 	github.com/gogo/protobuf => github.com/gravitational/protobuf v1.3.2-teleport.1
 	github.com/gravitational/teleport/api => ./api
 	github.com/julienschmidt/httprouter => github.com/gravitational/httprouter v1.3.1-0.20220408074523-c876c5e705a5
-	github.com/keys-pub/go-libfido2 => github.com/gravitational/go-libfido2 v1.5.3-0.20240123182224-399e6dce025f
+	github.com/keys-pub/go-libfido2 => github.com/gravitational/go-libfido2 v1.5.3-0.20240124203001-fa4d529ecc95
```

Comment:
I originally had no intention of managing tags for the go-libfido2 fork. From its [README][1]:

> The teleport branch is the default and reflects the library version used by Teleport. There is no concept of semantic versioning for the teleport branch, it rolls forward as Teleport needs it to.

This is partly because tags could potentially conflict with the upstream, and now partly because if we were to do this properly we would require a new major (as we've already landed >1 breaking change).

[1]: https://github.com/gravitational/go-libfido2?tab=readme-ov-file#why-fork

Reply:
I personally think tagging makes sense, it helps highlight from the fork which point a release occurred.

A tag on the fork should be completely isolated from the upstream module, so conflicts are not likely.  I have seen versions strategies where the modification version is appended (for example `v1.0.0` becomes `v1.0.0-1` or `v1.0.0-2` after subsequent changes).  That would completely avoid any confusion on where the tag belongs.

Reply:
OK, should we do `v1.5.3.-teleport.1`, so it matches the others?

Reply:
No opinions on the tag version, any tag will address my concerns.

Reply:
OK, please note that this PR needs https://github.com/gravitational/go-libfido2/pull/17, so I'd like to merge that before tagging. There are no more actions I can take here for the moment.

Reply:
This is now freshly rebased onto #37181, which is using the `v1.5.3-teleport.1` go-libfido2 tag.